client/app/components/show-project/actions-list/ulurp-number-link.js

```diff
@@ -0,0 +1,28 @@
+import Component from '@ember/component';
+import ENV from 'labs-zap-search/config/environment';
+
+/**
+ * @param {Action Object} action
+ */
+export default class UlurpNumberLinkComponent extends Component {
+  host = ENV.host;
+
+  ACTION_STATUSCODE_ACTIVE = 1;
```

Comment:
@allthesignals  I need to access this in the template though. Do you think it's better to have redundancy?
like 
```
ACTION_STATUSCODE_ACTIVE = 1;

export default class UlurpNumberLinkComponent extends Component {
    ACTION_STATUSCODE_ACTIVE = ACTION_STATUSCODE_ACTIVE;
```

Reply:
The convention I usually follow with that is to define a camel-case variable:

`actionStatuscodeActive = ACTION_STATUSCODE_ACTIVE;`

I have wanted a way to lookup action codes through a helper. I know you built something similar in applicant portal. 

But it might not matter if you end up refactoring the approach by retaining the helper.

Reply:
> The convention I usually follow with that is to define a camel-case variable:
> 
> `actionStatuscodeActive = ACTION_STATUSCODE_ACTIVE;`

Following up, my reasoning here is that ALL_CAPS usually indicates some kind of static, constant, easy to find, where camelCase is a class property.

Reply:
Gotcha. Will update